Firmware/fysetc-cheetah-v2.0.cfg

```diff
@@ -0,0 +1,350 @@
+# [mcu display]
+# serial: **PASTE YOUR SERIAL PORT HERE AND UNCOMMENT**
+# restart_method: command
+
+# [display]
+# lcd_type: sh1106
+# i2c_mcu: display
+# i2c_bus: i2c1a
+# # Set the direction of the encoder wheel
+# #   Standard: Right (clockwise) scrolls down or increases values. Left (counter-clockwise scrolls up or decreases values.
+# encoder_pins: ^display:PA3, ^display:PA4
+# #   Reversed: Right (clockwise) scrolls up or decreases values. Left (counter-clockwise scrolls down or increases values.
+# #encoder_pins: ^display:PA4, ^display:PA3
+# click_pin: ^!display:PA1
+# kill_pin: ^!display:PA5
+# #x_offset: 2
+# #   Use X offset to shift the display towards the right. Value can be 0 to 3
+# #vcomh: 0
+# #   Set the Vcomh value on SSD1306/SH1106 displays. This value is
+# #   associated with a "smearing" effect on some OLED displays. The
+# #   value may range from 0 to 63. Default is 0.
+# #   Adjust this value if you get some vertical stripes on your display. (31 seems to be a good value)
+
+# [neopixel displayStatus]
+# pin: display:PA0
+# chain_count: 1
+# color_order: GRB
+# initial_RED: 0.2
+# initial_GREEN: 0.05
+# initial_BLUE: 0
+
+#####################################################################
+# Macros
+#####################################################################
+[gcode_macro PRINT_START]
+#   Use PRINT_START for the slicer starting script - please customize for your slicer of choice
+gcode:
+    G28                            ; home all axes
+    G90                            ; absolute positioning    
+    G1 Z20 F3000                   ; move nozzle away from bed
+   
+[gcode_macro PRINT_END]
+#   Use PRINT_END for the slicer ending script - please customize for your slicer of choice
+gcode:
+
+    M400                           ; wait for buffer to clear
+    G92 E0                         ; zero the extruder
+    G1 E-4.0 F3600                 ; retract filament
+    G91                            ; relative positioning
+
+    #   Get Boundaries
+    {% set max_x = printer.configfile.config["stepper_x"]["position_max"]|float %}
+    {% set max_y = printer.configfile.config["stepper_y"]["position_max"]|float %}
+    {% set max_z = printer.configfile.config["stepper_z"]["position_max"]|float %}
+
+    #   Check end position to determine safe direction to move
+    {% if printer.toolhead.position.x < (max_x - 20) %}
+        {% set x_safe = 20.0 %}
+    {% else %}
+        {% set x_safe = -20.0 %}
+    {% endif %}
+
+    {% if printer.toolhead.position.y < (max_y - 20) %}
+        {% set y_safe = 20.0 %}
+    {% else %}
+        {% set y_safe = -20.0 %}
+    {% endif %}
+
+    {% if printer.toolhead.position.z < (max_z - 2) %}
+        {% set z_safe = 2.0 %}
+    {% else %}
+        {% set z_safe = max_z - printer.toolhead.position.z %}
+    {% endif %}
+
+    G0 Z{z_safe} F3600    ; move nozzle up
+    G0 X{x_safe} Y{y_safe} F20000    ; move nozzle to remove stringing
```

Comment:
Removed power off heater line see here for what the end macro should look like https://github.com/VoronDesign/Voron-0/blob/Voron0.2/Firmware/bigtreetech-skr-pico-v1.0.cfg#L304

Reply:
I create new PR for easier merge, check https://github.com/VoronDesign/Voron-0/pull/263